src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -708,8 +708,26 @@ protected void OnExited(object sender, EventArgs e)
                 //
                 stdInWriter.StopWriting();
             }
-            PSRemotingTransportException psrte = new PSRemotingTransportException(PSRemotingErrorId.IPCServerProcessExited,
-                                RemotingErrorIdStrings.IPCServerProcessExited);
+
+            int exitCode;
+            string exitMessage;
+            try
+            {
+                var jobProcess = (Process)sender;
+                exitCode = jobProcess.ExitCode;
+                exitMessage = exitCode == 0 ? jobProcess.StandardOutput.ReadToEnd() : jobProcess.StandardError.ReadToEnd();
```

Comment:
Is it possible for both stdout and stderror to have information on exit?  We may want to read both, but I think this is fine for now.

Reply:
Yeah I was thinking the same - wasn't sure what the policy is on cramming too much information into a single error message